src/RestServer/RestServerUtility.JTokens.cs

```diff
@@ -0,0 +1,302 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Plugins.RestServer is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Network.P2P.Payloads;
+using Neo.Network.P2P.Payloads.Conditions;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Newtonsoft.Json.Linq;
+
+namespace Neo.Plugins.RestServer
+{
+    public static partial class RestServerUtility
+    {
+        public static JToken BlockHeaderToJToken(Header header, global::Newtonsoft.Json.JsonSerializer serializer) =>
+            JToken.FromObject(new
+            {
+                header.Timestamp,
+                header.Version,
+                header.PrimaryIndex,
+                header.Index,
+                header.Nonce,
+                header.Hash,
+                header.MerkleRoot,
+                header.PrevHash,
+                header.NextConsensus,
+                Witness = WitnessToJToken(header.Witness, serializer),
+                header.Size,
```

Comment:
That is for `blockheader` **only**. I don't return any transactions. The reason for this follows, Saves network bandwidth if the requestor doesn't want transactions. So we only return block information for the current header. But I do see what your saying. How the _true_ block size with transactions is different compared to`blockheader`. It can be changed if you want.

Reply:
Then maybe it's better to rename it as `GetBlockHeaders` instead of `GetBlocks`.

Reply:
I see, just confused with some request name and result, will make suggestion below.